common/src/IO/ObjSerializer.cpp

```diff
@@ -90,7 +90,7 @@ namespace TrenchBroom {
         void ObjFileSerializer::writeTexCoords() {
             std::fprintf(m_stream, "# texture coordinates\n");
             for (const vm::vec2f& elem : m_texCoords.list()) {
-                std::fprintf(m_stream, "vt %.17g %.17g\n", static_cast<double>(elem.x()), static_cast<double>(elem.y()));
+                std::fprintf(m_stream, "vt %.17g %.17g\n", static_cast<double>(elem.x()), static_cast<double>(-elem.y()));
```

Comment:
Maybe add a comment explaining why this needs to be flipped - there's a chance that someone who looks at this 53 years from now will think that it was a typo ;-)